src/ir/BUILD

```diff
@@ -21,8 +22,22 @@ cc_test(
         ":ir",
         "//src/common/testing:gtest",
         "@absl//absl/hash",
+        "@absl//absl/hash:hash_testing",
```

Comment:
Not related to this CL. I wonder if we should gather all the `@absl` targets in a BUILD target.

Reply:
Perhaps. I doubt we're saving much in link time/binary size/whatever by keeping them separate, but are definitely encountering some review friction due to them. If we do this, we should probably make two targets, `absl` and `absl_testing`, so we don't mix our test and production dependencies. But having two libraries instead of many would be a big improvement.

Reply:
Filed #85 about this.